packages/design-system/src/components/autogrow-text-area.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * Implementation of the autogrowing text input for the AI
+ * https://www.figma.com/file/xCBegXEWxROLqA1Y31z2Xo/%F0%9F%93%96-Webstudio-Design-Docs?node-id=7586%3A48784&mode=dev
+ */
+
+import { type ComponentProps, type Ref, forwardRef } from "react";
+import { type CSS, css, theme } from "../stitches.config";
+import { textVariants } from "./text";
+import { Grid } from "./grid";
+import { useControllableState } from "@radix-ui/react-use-controllable-state";
+
+const commonStyle = css(textVariants.regular, {
+  border: "none",
+  paddingRight: 0,
+  paddingLeft: 0,
+  paddingTop: 0,
+  paddingBottom: 0,
+  boxSizing: "border-box",
+  gridArea: "1 / 1 / 2 / 2",
+  background: "transparent",
+  color: "inherit",
+  textWrap: "wrap",
+  overflowWrap: "break-word",
+  whiteSpace: "pre-wrap",
+  overflow: "hidden",
+});
+
+const style = css(commonStyle, {
+  outline: "none",
+  resize: "none",
+  "&::placeholder": {
+    color: theme.colors.foregroundContrastSubtle,
+  },
+  "&:disabled": {
+    color: theme.colors.foregroundDisabled,
+    background: theme.colors.backgroundInputDisabled,
+  },
+  variants: {
+    state: {
+      invalid: {
+        color: theme.colors.foregroundDestructive,
+      },
+    },
+  },
+});
+
+type Props = Omit<
+  ComponentProps<"textarea">,
+  "value" | "defaultValue" | "onChange"
+> & {
+  css?: CSS;
+  state?: "invalid";
+  value?: string;
+  defaultValue?: string;
+  onChange?: (value: string) => void;
+};
+
+export const AutogrowTextArea = forwardRef(
+  (
+    { css, className, state, value, defaultValue, onChange, ...props }: Props,
```

Comment:
```suggestion
    { css, className, state, value: initialValue, defaultValue, onChange, ...props }: Props,
```

Reply:
Its not initial value, it's value, and it's used as is if passed